src/module.model/_model.probes.ts

```diff
@@ -59,6 +62,15 @@ export class ModelProbeIndicator extends ProbeIndicator {
       return this.withDead('model', 'synced blocks are more than 2 blocks behind', details)
     }
 
+    // index defid can experience rollback, so make sure the condition is only checked if `Model == DeFid`
+    if (index === defid && now() - highest.time >= 90 * 60) {
```

Comment:
```suggestion
    if (index === defid && secondsSince(highest.time) >= 90 * 60) {
```

Minor suggestion for readability(?) haha